### PR TITLE
Zcu102 ci

### PR DESCRIPTION
**Pull request by chrisguikema**
Just trying to get CI testing for the CAmkES-VM. I'm open to other changes to get the testing up and running, just wanted to put something out there to test with.

### CODE REVIEW

**Comment by axel-h** [comment]
Making this a draft, as it has a hack on top that allow testing this in CI once before merging.

**Comment by lsf37** [comment]
Thanks for getting that started and tested out. I've made a smaller change over at #314 that should do the trick.

I'd like to avoid adding another copy for the platform, because that will affect all other CI jobs as well (e.g. sel4test would then also build a second set of images etc). Ideally we should sort out the name mismatch between the CAmkES VM cmake setup for `zcu102` and the rest of the build system for `zynqmp`, but we can do that separately.

File: seL4-platforms/platforms.yml [hunk 1]
```diff
@@ -213,6 +213,14 @@ platforms:
     march: armv7a
     disabled: true
 
+  ZCU102:
```
[the remark this answers is not in the document]
Comment (lsf37) [reply seL4-platforms/platforms.yml:216]: The rest of the build scripts all expect `zynqmp`, I expect this would break a lot of jobs.

**Comment by chrisguikema** [comment]
Thanks for doing that, Gerwin!

**Pull request closed without merging**